src/ModelHelpers/Attributes.php

```diff
@@ -1121,4 +1121,27 @@ public function hasField($key)
 
 		return in_array($key, $this->getDefaultFields());
 	}
+
+	/**
+	 * Get a nested attribute from the model.
+	 *
+	 * @param   string  $key attribute full name separated by . (SQL ordering friendly)
+	 *
+	 * @return mixed
+	 *
+	 * @throws AttributeNotFoundException
+	 */
+	public function getAttributeNested($key)
```

Comment:
I'm not sure this needs to be public - it's not something we really want people to use directly. It's just a helper function for the sorting closure